tests/results/scale/edge/edge-plus.md

```diff
@@ -148,21 +148,21 @@ The logs are attached only if there are errors.
 
 ### Event Batch Processing
 
-- Total: 104
-- Average Time: 117ms
+- Total: 43
```

Comment:
That's an interesting one

Reply:
Yeah, I think we need to make sure our metrics collection is accurate.